pkgs/development/tools/build-managers/jam/default.nix

```diff
@@ -1,11 +1,5 @@
 {stdenv, fetchurl, yacc}:
 
-let
-  bindir = if stdenv.system == "i686-linux" then "bin.linuxx86"
-    else if stdenv.system == "x86_64-linux" then "bin.linux"
-    else throw "Unsupported platform by now";
-in
-
 stdenv.mkDerivation {
   name = "jam-2.5";
   src = fetchurl {
```

Comment:
url = ftp://ftp.perforce.com/jam/${name}.tar;
Makes it easier for when version changes; change one version area verses two